contrib/init/sysvinit-debian/docker

```diff
@@ -39,18 +39,19 @@ if [ -f /etc/default/$BASE ]; then
 	. /etc/default/$BASE
 fi
 
-# see also init_is_upstart in /lib/lsb/init-functions (which isn't available in Ubuntu 12.04, or we'd use it)
-if [ -x /sbin/initctl ] && /sbin/initctl version 2>/dev/null | grep -q upstart; then
-	log_failure_msg "$DOCKER_DESC is managed via upstart, try using service $BASE $1"
-	exit 1
-fi
-
 # Check docker is present
 if [ ! -x $DOCKER ]; then
 	log_failure_msg "$DOCKER not present or not executable"
 	exit 1
 fi
 
+check_init() {
+         if init_is_upstart; then
```

Comment:
```init_is_upstart``` comes from ```/lib/lsb/init-functions``` which we reference on https://github.com/stevenbrichards/docker/blob/13031-Upstart/contrib/init/sysvinit-debian/docker#L35-L36